spec/valkyrie/persistence/active_fedora/persister_spec.rb

```diff
@@ -4,5 +4,5 @@
 
 RSpec.describe Valkyrie::Persistence::ActiveFedora::Persister do
   let(:persister) { described_class }
-  it_behaves_like "a Valkyrie::Persister"
+  it_behaves_like "a Valkyrie::Persister", :no_deep_nesting, :no_mixed_nesting
```

Comment:
If we're going to merge this "get AF adapter to support deep nesting and mixed nesting" is definitely a ticket we should open.